internal/runtimehandlerhooks/high_performance_hooks.go

```diff
@@ -64,6 +105,24 @@ func shouldCPULoadBalancingBeDisabled(annotations fields.Set) bool {
 	return value == "true"
 }
 
+func shouldCPUQuotaBeDisabled(annotations fields.Set) bool {
+	value, ok := annotations[annotationCPUQuota]
+	if !ok {
+		return false
+	}
+
+	return value == "true"
```

Comment:
You can avoid checking `ok` since for non-existing map elements the value returned is `""`.

Reply:
I.e. 
```go
return annotations[annotationCPUQuota] == "true"
```
is both correct and sufficient.